app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -17,6 +18,44 @@
   color: govuk-colour("white");
 }
 
-.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
+.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:after {
   border-color: govuk-colour("white");
 }
+
+.govuk-breadcrumbs .govuk-breadcrumbs__list-item {
```

Comment:
I don't think this needs to be nested like this - will it work as `.govuk-breadcrumbs__list-item {`?

Reply:
Removed this and is fine